sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2867,6 +2867,25 @@ object functions {
     new Overlay(src.expr, replace.expr, pos.expr)
   }
 
+  /**
+   * Splits a string into arrays of sentences, where each sentence is an array of words.
+   * @group string_funcs
+   * @since 3.2.0
+   */
+  def sentences(str: Column, language: String, country: String): Column = withExpr {
```

Comment:
I think we should make it `Column` for both `language` and `country` in case they we make it accept non-literal values (which is also documented on the top of this file).